To-do-list/Model/TaskManager.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  TaskManager.swift
+//  To-do-list
+//
+//  Created by ineta.magone on 03/03/2022.
+//
+
+import UIKit
+
+class TaskManager {
+    
+    // MARK: - Variables
+    
+    var titleArray = [String]()
+    var descriptionArray = [String]()
+ 
+    // MARK: - Methods
+    
+    // ViewController
+    func reloadTableView(tableView: UITableView) {
```

Comment:
How will you work with this function at future?

Reply:
I wanted to use it on ViewController after coming back from TaskViewController. For the moment didn't find way to implement it correctly.